<?xml version="1.0"?>
<launch>
    <node name="perception" pkg="perception" type="perception" output="screen">
        <param name="image_topic_prefix" value="/carla/camera_"/>
        <param name="batch_size" type="int" value="4"/>
    </node>
</launch>